### ISSUE TITLE
Move React on top 

### ISSUE THREAD

**Issue by Deliaz** [trivago/prettier-plugin-sort-imports#7]
Thanks for the plugin. The default order works for me, except one thing: Is there a way to move React on top, and keep it as a first-line?

**Comment by kettanaito** [comment]
That should be possible by providing React's RegExp in the `importOrder` of the Prettier config.

```js
{
  "importOrder": ["react"]
}
```

**Comment by Deliaz** [comment]
@kettanaito yeah, but with `importOrderSeparation: true` it will add another new line, separating 'react' into a new group. 
Is it designed behavior?

**Comment by nicolad** [comment]
agree, we also use React at the top, but w/o any separation from other global imports.
@ayusharma do you think it's possible to achieve this?

**Comment by moshfeu** [comment]
For how is interested, in order to order react (and its associated libraries such as react-router-dom), then other modules then internal improts

```json
["react", "^(?!react)\\w+$", "^[./]"]
```

**Comment by ayusharma** [comment]
Yes. You can use the import order to make react on top.

**Comment by ayusharma** [comment]
You can not move react on top. The plugin's idea is to just keep a separation between the third party and local imports.
